sycl/source/detail/jit_compiler.cpp

```diff
@@ -664,14 +664,22 @@ ur_kernel_handle_t jit_compiler::materializeSpecConstants(
   AddToConfigHandle(
       ::jit_compiler::option::JITEnableVerbose::set(DebugEnabled));
 
-  std::string TargetCPU =
-      detail::SYCLConfig<detail::SYCL_JIT_AMDGCN_PTX_TARGET_CPU>::get();
-  std::string TargetFeatures =
-      detail::SYCLConfig<detail::SYCL_JIT_AMDGCN_PTX_TARGET_FEATURES>::get();
+  auto SetUpOption = [](std::string Value) {
+    ::jit_compiler::JITEnvVar Option(Value.size());
+    memcpy(static_cast<void *>(Option.begin()),
+           static_cast<const void *>(Value.data()), Value.size());
+    return Option;
```

Comment:
```suggestion
    return ::jit_compiler::JITEnvVar{Value.size(), Value.data()};
```

And implement constructor in `DynArray`